examples/controls_example/public/react_controls/data_controls/options_list_control/components/options_list_editor_options.tsx

```diff
@@ -0,0 +1,175 @@
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useEffect, useMemo, useState } from 'react';
+
+import { EuiFormRow, EuiRadioGroup, EuiSwitch } from '@elastic/eui';
+import { useStateFromPublishingSubject } from '@kbn/presentation-publishing';
+
+import {
+  getCompatibleSearchTechniques,
+  OptionsListSearchTechnique,
+} from '../../../../../common/options_list/suggestions_searching';
+import { ControlSettingTooltipLabel } from '../../../components/control_setting_tooltip_label';
+import { CustomOptionsComponentProps } from '../../types';
+import { DEFAULT_SEARCH_TECHNIQUE } from '../constants';
+import { OptionsListControlState } from '../types';
+import { OptionsListStrings } from '../options_list_strings';
+
+const selectionOptions = [
+  {
+    id: 'multi',
+    label: OptionsListStrings.editor.selectionTypes.multi.getLabel(),
+    'data-test-subj': 'optionsListControl__multiSearchOptionAdditionalSetting',
+  },
+  {
+    id: 'single',
+    label: OptionsListStrings.editor.selectionTypes.single.getLabel(),
+    'data-test-subj': 'optionsListControl__singleSearchOptionAdditionalSetting',
+  },
+];
+
+const allSearchOptions = [
+  {
+    id: 'prefix',
+    label: (
+      <ControlSettingTooltipLabel
+        label={OptionsListStrings.editor.searchTypes.prefix.getLabel()}
+        tooltip={OptionsListStrings.editor.searchTypes.prefix.getTooltip()}
+      />
+    ),
+    'data-test-subj': 'optionsListControl__prefixSearchOptionAdditionalSetting',
+  },
+  {
+    id: 'wildcard',
+    label: (
+      <ControlSettingTooltipLabel
+        label={OptionsListStrings.editor.searchTypes.wildcard.getLabel()}
+        tooltip={OptionsListStrings.editor.searchTypes.wildcard.getTooltip()}
+      />
+    ),
+    'data-test-subj': 'optionsListControl__wildcardSearchOptionAdditionalSetting',
+  },
+  {
+    id: 'exact',
+    label: (
+      <ControlSettingTooltipLabel
+        label={OptionsListStrings.editor.searchTypes.exact.getLabel()}
+        tooltip={OptionsListStrings.editor.searchTypes.exact.getTooltip()}
+      />
+    ),
+    'data-test-subj': 'optionsListControl__exactSearchOptionAdditionalSetting',
+  },
+];
+
+export const OptionsListEditorOptions = ({
+  initialState,
+  field,
+  updateState,
+  parentApi,
+}: CustomOptionsComponentProps<OptionsListControlState>) => {
+  const allowExpensiveQueries = useStateFromPublishingSubject(parentApi.allowExpensiveQueries$);
+
+  const [singleSelect, setSingleSelect] = useState<boolean>(initialState.singleSelect ?? false);
+  const [runPastTimeout, setRunPastTimeout] = useState<boolean>(
+    initialState.runPastTimeout ?? false
+  );
+  const [searchTechnique, setSearchTechnique] = useState<OptionsListSearchTechnique>(
```

Comment:
Same comment, do not set to default value. Set default value on read

Reply:
See https://github.com/elastic/kibana/pull/186655#discussion_r1697305846